Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM --platform=linux/amd64 python:3.10.6-buster
+
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+ENV PYTHONPATH /
+
```

Comment:
Also docker-compose up is saying it's finding orphan containers which I think could just be from my docker having a project with the same name.